contracts/governance/FeeSharingProxy/FeeSharingLogic.sol

```diff
@@ -99,6 +109,44 @@ contract FeeSharingLogic is SafeMath96, IFeeSharingProxy, Ownable, FeeSharingPro
 		emit FeeWithdrawn(msg.sender, loanPoolToken, poolTokenAmount);
 	}
 
+	/**
+	 * @notice Withdraw amm fees for the given converter addresses:
+	 * protocolFee from the conversion
+	 * the fees will be converted in wRBTC form, and then will be transferred to wRBTC loan pool
+	 *
+	 * @param _converters array addresses of the converters
+	 * */
+	function withdrawFeesAMM(address[] memory _converters) public {
+		address wRBTCAddress = protocol.wrbtcToken();
+		require(wRBTCAddress != address(0), "FeeSharingProxy::withdrawFees: wRBTCAddress is not set");
+
+		address loanPoolToken = protocol.underlyingToLoanPool(wRBTCAddress);
+		require(loanPoolToken != address(0), "FeeSharingProxy::withdrawFees: loan wRBTC not found");
+
+		for (uint256 i = 0; i < _converters.length; i++) {
+			require(Address.isContract(_converters[i]), "FeeSharingProxy::withdrawFees: converter is not a contract");
+
+			uint256 wrbtcAmountWithdrawn = IConverterAMM(_converters[i]).withdrawFees(address(this));
```

Comment:
Anyone can call this function and pass anything as a converter address.
Attacker can use a contract that returns a positive `wrbtcAmountWithdrawn` value, but does not transfer any assets.
If the contract holds wrbtc from any other source, these tokens would be incorrectly sent to the LoanToken

Reply:
Whitelist mechanism for converter has been added